zkevm-circuits/src/exp_circuit/param.rs

```diff
@@ -1,5 +1,5 @@
 /// The number of rows assigned for each step in an exponentiation trace.
-pub(crate) const OFFSET_INCREMENT: usize = 7usize;
+pub(crate) const OFFSET_INCREMENT: usize = 8usize;
```

Comment:
for new padding row,  need to constrain value is zero, otherwise can fill with any value.